pro_tes/middleware/middleware.py

```diff
@@ -35,7 +44,7 @@ def modify_request(self, request):
             request: Incoming request object.
 
         Returns:
-            Tuple of modified request object.
+            The modified request object.
         """
```

Comment:
Make sure that in all methods/functions that _explicitly_ raise an error, there is a `Raises:` section that describes what error is raised and under what circumstances. Here, for example:

```
"""...

Args:
    ...

Returns:
    ...

Raises:
    `pro_tes.exceptions.NoTesInstancesAvailable`: If no valid TES instances are
        available.
"""
```

Reply:
Done